packages/prisma/schema.prisma

```diff
@@ -139,6 +139,8 @@ model EventType {
 
   secondaryEmailId Int?
   secondaryEmail   SecondaryEmail? @relation(fields: [secondaryEmailId], references: [id], onDelete: Cascade)
+  //This is used to pass session dependent actorUserId for AuditLogger Client Extension. It is always null.
```

Comment:
What do you mean by "It is always null"?

Reply:
It is always holds null it doesn't hold any value the reason I used it is for this feature is to hold the session dependent actorUserId. In other words, it holds who invoked the query. But after getting the actorUserId on the client extension and registering it on the auditlog it will have to value so it is better to nullify it.